mmdet/models/dense_heads/solo_head.py

```diff
@@ -135,13 +135,18 @@ def resize_feats(self, feats):
         for i in range(len(feats)):
             if i == 0:
                 out.append(
-                    F.interpolate(feats[0], scale_factor=0.5, mode='bilinear'))
+                    F.interpolate(
+                        feats[0],
```

Comment:
This may get some error if use scale_fctor. Because if the aaa cannot divide by 2, after resize, maybe it can't match the h and w?

Reply:
Yep. Personally, I think this change here in SOLO head actually makes little difference to the model's performance because there is no interaction across feature levels, and the code would work even when the feature map's sizes do not match across different levels. The only reason I did the change here is to make the logic looks better and nothing else ;-)

BTW, things are different in the code of MaskFeatureHead of SOLOv2. SOLOv2 uses unified MaskFeatureHead which uses element-add to fuse the features across levels. Thus the upsample and downsample output feature map shape must match the shape of the neighboring level.